hexa/databases/schema.py

```diff
@@ -38,6 +41,42 @@ def resolve_database_table(workspace, info, **kwargs):
     return get_table_definition(workspace, kwargs.get("name"))
 
 
+@database_object.field("credentials")
```

Comment:
@cheikhgwane this will work but it would be easier to return a simple dict (instead of returning the workspace and then writing a bunch of resolvers) with all the connection parameters.

Reply:
Also, could we encapsulate the different connection parameters in the workspace model? (thinking about port / url / host etc). They could be defined as @property methods.

Reply:
Like the username, define properties like `workspace_db_host`?

Reply:
`db_host`, `db_port`, `db_url`... That way the logic is encapsulated in the workspace and does not need to be copied / pasted elsewhere.

Reply:
Yes, make sense.